pwndbg/commands/heap.py

```diff
@@ -22,6 +22,15 @@ def value_from_type(type_name, addr):
     gdb_type = pwndbg.typeinfo.load(type_name)
     return gdb.Value(addr).cast(gdb_type.pointer()).dereference()
 
+def read_chunk(addr):
+    renames = {
+        "mchunk_size": "size",
+        "mchunk_prev_size": "prev_size",
+    }
+    val = value_from_type("struct malloc_chunk", addr)
+    return dict({ renames.get(key) or key: int(val[key]) for key in val.type.keys() }, value=val)
```

Comment:
The `{ renames.get(key) or key: int(val[key])` looks weird/wrong.

The whole line could be simplified to:
```python
return dict({renames.get(key, key): int(val[key]) for key in val.type.keys()}, value=val)
```